lfds-nswag.json

```diff
@@ -0,0 +1,22 @@
+{
+  "runtime": "Default",
+  "defaultVariables": null,
+  "documentGenerator": {
+    "fromDocument": {
+      "url": "lfds-swagger.json",
+      "output": null,
+      "newLineBehavior": "Auto"
+    }
+  },
+  "codeGenerators": {
+    "openApiToCSharpClient": {
+      "classname": "{controller}ApiClient",
```

Comment:
could you change this to `"{controller}Client"`. I'll be updating the `TokenApiClient` to `TokenClient` in a PR soon to match the naming convention in the js library.